README.md

```diff
@@ -37,7 +37,9 @@ If you already have Node.js installed, an easy way to install `n` is using `npm`
 
     npm install -g n
 
-Once installed, `n` caches Node.js versions in subdirectory `n/versions` of the directory specified in environment variable `N_PREFIX`, which defaults to `/usr/local`; and the _active_ Node.js version is installed under `N_PREFIX`.
+Once installed, `n` caches Node.js versions in subdirectory `n/versions` of the directory specified in environment variable `N_PREFIX`, which defaults to `/usr/local`.
+
+The _active_ Node.js version is installed under `N_PREFIX`. (This creates subdirectories `bin`, `include`, `lib` and `share`.)
```

Comment:
Perhaps "uses" rather than "creates"? In the default location the subdirectories already exist.

Reply:
In my current understanding, installing version does creates new folders under N_PREFIX location

And if N_PREFIX points to some system folder, then installing will fail without `sudo`